utensor_cgen/optimizer.py

```diff
@@ -0,0 +1,40 @@
+#-*- coding:utf8 -*-
+from collections import defaultdict
+
+
+class Optimizer(object):
+
+  @classmethod
+  def optimize(cls, ops_info, bfs_order, output_nodes, level='simple'):
```

Comment:
Need a default `level=None` for no optimization at all. Pretty sure this is handled by simple

Reply:
You mean `level=None` for no optimization (no ref counting, ...etc)
And maybe `level='simple'` means simple ref counting?

Reply:
yes

Reply:
Ok, working on it

Reply:
cool, I think it's ok to just note this for now and add it as a feature request in the issues tab.